File: Student.cpp
#include "Student.h"

Student::Student(string name, long id, int age, int avg=0, string institute="None") :Person(name, id, age) {
	if (avg >= 0)
		this->avg = avg;
	else
		cout << "Average cannot be negative! set to zero.\n";
	 
	this->institute = institute;
}

void Student::print()const{
	Person::print();
	cout << "Average: " << avg << endl
		<< "Institute: " << institute << endl;
}

File: Student.py
from Person import Person

class Student(Person):

    def __init__(self, name, id, age, average, institute):
        Person.__init__(self, name, id, age)
        self.average = float(average)
        self.institute = institute
        while(len(self.institute) > 10):
            self.institute = input("Enter shorter instiute's name:")
        self.institute = self.institute.title()

    def __str__(self):
        #print operator
        print("-----------")
        return f'{Person.__str__(self)}Institute: {self.institute}\nAverage: {self.average}\n'




